src/main/java/blue/lapis/pore/impl/entity/PoreTameable.java

```diff
@@ -28,40 +28,48 @@
 
 import org.bukkit.entity.AnimalTamer;
 import org.bukkit.entity.Creature;
-import org.spongepowered.api.entity.living.Tameable;
+import org.spongepowered.api.data.manipulators.entities.TameableData;
 import org.spongepowered.api.entity.living.animal.Animal;
 
 public class PoreTameable extends PoreAnimals implements org.bukkit.entity.Tameable, Creature {
 
-    public static PoreTameable of(Tameable handle) {
+    public static PoreTameable of(Animal handle) {
         return WrapperConverter.of(PoreTameable.class, handle);
     }
 
-    protected PoreTameable(Tameable handle) {
-        super((Animal) handle);
+    protected PoreTameable(Animal handle) {
+        super(handle);
     }
 
-    private Tameable getTameable() {
-        return (Tameable) getHandle();
+    @SuppressWarnings("unused")
+    private Animal getTameable() {// TODO: is this being used?
```

Comment:
Seeing as the Data API removed Tameable, no, it's not necessary anymore. Previously it was just a convenience method to avoid explicit typecasting.